source/includes/_intro.md

```diff
@@ -394,25 +394,267 @@ The first step in most situations will be to instantiate a client. (TODO these c
 
-(TODO also include how to install eg with pip install?)
-
+(TODO also include how to install e.g. with pip install?)
+
+
+## Required Minimum Functionality
+
+Before your application can go live, there are a number of minimum
+implementation requirements it must meet. The test data available can help make
+sure your implementation is working as expected and can handle various types of
+errors and edge cases. To validate the functionality of your application,
+Ingresso will use the following representative tests.
+
+### List performance dates or times
+A given event may have multiple performances on different days and times, and
+the customer must be able to choose the one they want to attend. This can be
+done by listing them, or in a calendar interface, or in any other way that is
+appropriate for your application.
+
+**Representative Test:** User selects *The Unremarkable Incident of the Cat at Lunchtime*
+(event ID 7AB)<br/>
+When the event is selected in the interface,<br/>
+Then it should display the two available performances, 1 January {this year + 1}
+and 1 January {this year + 2}, at 15:30:00 UTC.
+
+### Support best available booking flow
+At a minimum, the customer must be able to specify the number of seats they
+want and for which ticket type and price band. The Ingresso backend will then
+reserve the best available seats that meet the requirements.
+
+**Representative Test:** User selects 3 tickets for event 7AB in the stalls band B<br/>
+When the reservation is made<br/>
+Then the application shows a reservation confirmation screen with 3 tickets from
+that price band (such as D2, D3 and D4), and allows the user to proceed with
+the booking and payment process
+
+### Display the full ticket price to the customer before purchase
+The customer must be shown the full price they will pay before the purchase is
+confirmed and their payment taken.
+<aside class="notice">If selling in the UK market, the ticket seat price and
+the surcharge should be itemised separately. Make sure to follow all market
+regulations for the countries and regions you are selling in.</aside>
+
+**Representative Test:** User confirms reservation of 1 seat for event 7AB in the stalls
+band A (e.g. seat B4)<br/>
+When the confirmation and payment screen is displayed<br/>
+Then the full price should be displayed as £55.00, showing a £50.00 seat price
+and a £5.00 surcharge. Any dispatch methods that incur an additional cost
+should also be listed separately. A total price should be displayed in a
+highlighted font showing the price the customer will pay (by adding the prices
+together).
+
+### Display special seat conditions to customer before purchase
+Some seats in venues may have restricted views of the stage, or other
+information associated with them. It's important that the customers are made
+aware of any special seat conditions that will apply before they purchase.
+
+**Representative Test:** User confirms reservation of 3 seats in CIRCLE-B for
```

Comment:
I suggest to have a test for both best available and seat selection. So leave what you have, but make that optional for partners implementing their own seat selection (or move it into the section below).

Then add an example for best available restricted view.